### DISCUSSION TITLE
Will sx1262/1268 be supported

### DISCUSSION THREAD

**Discussion by StreakingJerry** [ExpressLRS/ExpressLRS#1718] [Q&A]
The new sx1262/1268 is muck better than sx1272. Will the modual such as Ra-01s and E22-400M30S be supported? (they are 433 mhz which can be even longer range and won't confilct with cellphone network band 8)

**Comment by FelixvdDonk** [comment]
Any word on this? Also I'd like to mention that 433 is already possible with the SX1278, I tried it before

**Comment by Tanmay-FuelDaddy** [comment]
Awaiting eagerly for the same, Have some great ideas involving Multi Rx setup on a Rover

**Comment by JyeSmith** [comment]
The SX1262/LLCC68 do not perform as well as the SX1276.  The link below has info that was posted on the now closed Semtech forum.

433 is being tested, but to date has not performed as well as 900M.  This result has been consistent across multiple Semtech transceivers. So there is no officially support commercial 433 hardware from ELRS at this stage.

https://web.archive.org/web/20240310140117/https://forum.lora-developers.semtech.com/t/llcc68-poor-sensitivity/1480
